docs/2.5. APIs - Query Parameters.md

```diff
@@ -81,31 +86,35 @@ Payload Resources
 ```
 
 ***Notes***
-* The data set returned when no query parameters are specified must be from the most recently updated (or created) resources in the collection, returned in descending order.
-* The X-Paging headers identify properties of the collection of data returned in the response. These parameters may be used to construct a URL which would return the same set of bounded data on consecutive requests (for example /x-nmos/query/v1.1/nodes?paging.since=0:10&paging.until=0:20).
-* The Link header identifies the 'next' and 'prev' cursors which an application may use to make its next requests. An implementation may also provide 'first' and 'last' cursors to identify the beginning and end of a collection if this can be addressed via a consistent URL.
+
+- The data set returned when no query parameters are specified MUST be from the most recently updated (or created) resources in the collection, returned in descending order.
+- The X-Paging headers identify properties of the collection of data returned in the response. These parameters MAY be used to construct a URL which would return the same set of bounded data on consecutive requests (for example /x-nmos/query/v1.1/nodes?paging.since=0:10&paging.until=0:20).
+- The Link header identifies the 'next' and 'prev' cursors which an application MAY use to make its next requests. An implementation MAY also provide 'first' and 'last' cursors to identify the beginning and end of a collection if this can be addressed via a consistent URL.
```

Comment:
Not sure if that first "MAY" is really a normative requirement, it might be better as "could" or "can"?

Reply:
It's not really about granting permission so "can" better.

Reply:
This conversation was marked as resolved, but I don't see an actual commit or suggestion to change 'MAY' to 'can'? Let's discuss this one tomorrow.